src/components/Login.tsx

```diff
@@ -1,9 +1,135 @@
- const Login:React.FC = () => {
-    return (
-        <div>
-            <p>Login</p>
-        </div>
-    )
-}
-
-export default Login;
\ No newline at end of file
+import { useRef, useState } from "react";
+import { checkValidEmail } from "./utils/validate";
+import HeaderComponent from "./ui/HeaderComponent";
+import { Link } from "react-router-dom";
+import { auth } from "./utils/firebase";
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword } from "firebase/auth";
```

Comment:
Usually the import order is in format of: 
- main framework imports (react, react-dom)
- third party lib (zustand, antd, lodash, firebase)
- folder hierarchy based on levels
- style imports

Reply:
updated